test/models/customer_test.rb

```diff
@@ -0,0 +1,84 @@
+require "test_helper"
+
+describe Customer do
+  # let(:customer) { Customer.new }
+
+  describe "relationships" do
+    it "can have many rentals" do
+      customer = customers(:one)
+      customer.rentals.count.must_equal 3
+    end
+
+    it "returns an empty array if there are no rentals created under this customer" do
+      customer = Customer.create(name: "who", phone: "123-1234")
+      arry = []
+      customer.rentals.must_equal arry
+    end
+
+    it "returns an empty array for movies if there are no rentals created under this customer" do
+      customer = Customer.create(name: "who", phone: "123-1234")
+      arry = []
+      customer.movies.must_equal arry
+    end
+
+    it "can have many movies" do
+      customer = customers(:one)
+      customer.movies.count.must_equal 3
+    end
```

Comment:
This seems strange to me. What does it mean to "have a movie" if not to rent it?